humanguide/design-concepts/app-icon-design.rst

```diff
@@ -0,0 +1,109 @@
+App icon design
+===============
+
+The whole system and apps follow the `Suru design principles <http://web.archive.org/web/20191223082720/https://docs.ubuntu.com/phone/en/apps/design/get-started/design-philosophy>`__ started by Canonical and `Samuel Hewitt <https://icons.samuelhewitt.com/>`__.
```

Comment:
It may be better to link to Samuel's homepage, https://samuelhewitt.com/

Reply:
- I linked Samuel's specific section in his name
- Old Canonical website has more information and it's a bit more complete.I think is better for history reasons (for now)

Reply:
Yes, but you linked to icons.samualhewitt.com, not the homepage

Reply:
I see